src/poly.jl

```diff
@@ -76,7 +82,18 @@ for TPoly in [:ArbPoly, :AcbPoly]
     end
 end
 
-for (TSeries, T) in [(:ArbSeries, :Arb), (:AcbSeries, :Acb)]
+for (TSeries, TPolyStruct, T) in
```

Comment:
does `T` occur in the body of the loop?

Reply:
It does not! I must have removed that part in a earlier commit. Related to this:

Instead of specifying `TPolyStruct` I could do `TPolyStruct = cstructtype($TSeries)` inside the loop. This would probably be cleaner, but I was worried it would generate different code in the end, should maybe try it.

Reply:
Seems like it worked well, see my latest commit.